lib/segment/tests/integration/sparse_vector_index_search_tests.rs

```diff
@@ -22,10 +22,10 @@ use sparse::index::inverted_index::InvertedIndex;
 use tempfile::Builder;
 
 /// Max dimension of sparse vectors used in tests
-const MAX_SPARSE_DIM: usize = 2048;
+const MAX_SPARSE_DIM: usize = 4096;
```

Comment:
Took the opportunity to increase the size of the test data set given the recent performance improvements.